Switch to a branch packaged in Fedora, also seen in upstream commit
72ac94b2637b73f23ef98b2f6fe9c622ad4287fc.

--- a/src/vmm/Cargo.toml
+++ b/src/vmm/Cargo.toml
@@ -10,7 +10,7 @@
 derive_more = { version = "0.99.17", default-features = false, features = ["from", "display"] }
 event-manager = "0.3.0"
 kvm-bindings = { version = "0.6.0", features = ["fam-wrappers"] }
-kvm-ioctls = "0.12.0"
+kvm-ioctls = "0.14.0"
 lazy_static = "1.4.0"
 libc = "0.2.117"
 linux-loader = "0.8.1"
